holoviews/tests/plotting/bokeh/test_rasterplot.py

```diff
@@ -150,3 +151,138 @@ def test_rgb_invert_yaxis(self):
             self.assertEqual(cdata['y'], [-0.5])
         else:
             self.assertEqual(cdata['y'], [0.5])
+
+    def test_image_stack_tuple(self):
+        x = np.arange(0, 3)
+        y = np.arange(5, 8)
+        a = np.array([[np.nan, np.nan, 1], [np.nan] * 3, [np.nan] * 3])
+        b = np.array([[np.nan] * 3, [1, 1, np.nan], [np.nan] * 3])
+        c = np.array([[np.nan] * 3, [np.nan] * 3, [1, 1, 1]])
+
+        img_stack = ImageStack((x, y, a, b, c), kdims=["x", "y"], vdims=["a", "b", "c"])
+        plot = bokeh_renderer.get_plot(img_stack)
+        source = plot.handles['source']
+        self.assertEqual(source.data['image'][0][0], a.T)
```

Comment:
At some point, I want to transition to using pytest. 

`self.assertEqual` has special power in the holoviews testing framework, e.g., deep comparison of elements. 

For these though it seems like it is just a "simple" comparison that can be done with `np.testing.assert_array_almost_equal`.